### ISSUE TITLE
Manage it locally to share it globally: RDM and Wikimedia Commons

### ISSUE THREAD

**Issue by libnsh** [open-science-promoters/RDM-promotion#17]
Status: In github
Date: 21/08/2018
Your name: Nick Sheppard
Your orcid: https://orcid.org/0000-0002-3400-0274
license to apply to your comment: CC0
your position at the time your life story happened: NA
Input = project

The very first [Data Management Engagement Award](http://www.rdmengagementaward.org/), a competition sponsored by SPARC Europe, the University of Cambridge and Jisc to elicit new and imaginative ideas for engaging researchers in the practices of good Research Data Management (RDM).

Our proposal is to link RDM with the open science movement via the Wikimedia suite of tools.

Basically just share yours or others’ openly licensed research material via Wikimedia Commons to create an aggregated resource that can be used to improve Wikipedia. It’s easy!

To read more about the project and register your interest please visit [this shared Google folder](https://drive.google.com/drive/folders/1-bnrdFS4dq-IQ-7Ry9jAzZlhZvxKm4JL) which includes the original proposal and a preliminary project plan.

**Comment by jcolomb** [comment]
Thank you for the input, I will add it with the strategies tag.

good luck with the project, keep us posted on your successses.

**Comment by jcolomb** [comment]
Would you like to be added to the list of  team members ?

**Comment by libnsh** [comment]
Thanks Julien - glad I found your project!
